src/pages/VaccinePage.js

```diff
@@ -55,13 +57,21 @@ export const VaccinePageComponent = ({
     []
   );
 
-  return fridges.length ? <FlatList data={fridges} renderItem={Fridge} /> : <BlankComponent />;
+  return (
+    <DataTablePageView>
+      <FlexView flex={0} alignItems="flex-end" style={{ margin: 10 }}>
+        <PageButton text={generalStrings.admin} style={{}} onPress={onViewAdminPage} />
```

Comment:
I think `style` redundant here?

```suggestion
        <PageButton text={generalStrings.admin} onPress={onViewAdminPage} />
```